### ISSUE TITLE
MOC and ACC/Gyres response

### ISSUE THREAD

**Issue by PaulSpence** [pedrocol/basal_mom5-collaborative-project#89]
Control MOC and basal melt parameterized anomalies averaged over last 3 years (8-10). Not much of an integrated MOC response.

![Screenshot 2024-10-03 at 12 52 18 PM](https://github.com/user-attachments/assets/5241ce4f-a8ab-4a20-b5b3-101fe3363da8)

**Comment by adele-morrison** [comment]
Yeah, I guess it hasn't been run for long enough to see effects very far north. Probably not worth including in the paper I would say. Just stick to the cross-1000m isobath transports. What do you think?

**Comment by PaulSpence** [comment]
Big in time averaging is fixed in image ... slightly larger response ... ~2Sv

**Comment by adele-morrison** [comment]
From these it looks like the lower cell increases in Basal Gade (consistent with cross-1000m DSW transport), but decreases in Basal Gade Brine (inconsistent with [cross-1000m DSW transport](https://github.com/pedrocol/basal_mom5-collaborative-project/issues/65#issuecomment-2367020408)). Unless we can explain why these MOC changes are the way they are, I think they are going to confuse the story rather than help explain what is going on, so would advocate not including them. I don't think it would be too hard to justify leaving them out given the short run duration.

But maybe someone else can explain them?

**Comment by fabiobdias** [comment]
It seems to me there’s a slight increase in the lower cell in the BasalGadeBrine (south of 70°S) and a stronger decrease north of 70°S (similar to BasalGade), so the south bit is actually in agreement with the cross-1000m DWS transport. I don’t understand the changes in the deep cell thought.

But still, I think Claire’s figure summarise this all, so I also vote to not include this one.

From: Adele Morrison ***@***.***>
Date: Thursday, 3 October 2024 at 3:16 PM
To: pedrocol/basal_mom5-collaborative-project ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [pedrocol/basal_mom5-collaborative-project] MOC and ACC/Gyres response (Issue #89)

From these it looks like the lower cell increases in Basal Gade (consistent with cross-1000m DSW transport), but decreases in Basal Gade Brine (inconsistent with cross-1000m DSW transport<https://github.com/pedrocol/basal_mom5-collaborative-project/issues/65#issuecomment-2367020408>). Unless we can explain why these MOC changes are the way they are, I think they are going to confuse the story rather than help explain what is going on, so would advocate not including them. I don't think it would be too hard to justify leaving them out given the short run duration.

But maybe someone else can explain them?

**Comment by PaulSpence** [comment]
Barotropic Stream function response averaged over last 3 years:

![Screenshot 2024-10-31 at 3 04 55 PM](https://github.com/user-attachments/assets/4a94b4c2-eb07-4c75-9645-3d8bf931e345)

~4-8Sv increase in DP transport in Basal Gade runs

**Comment by adele-morrison** [comment]
Yes that fits with the size of the [changes in dense water crossing the 1000m isobath](https://github.com/pedrocol/basal_mom5-collaborative-project/issues/65#issuecomment-2367020408). More bottom water production = higher ACC transport.

**Comment by PaulSpence** [comment]
<img width="586" alt="Screenshot 2024-11-07 at 10 14 42 AM" src="https://github.com/user-attachments/assets/a9824fde-fc31-4b3d-95c4-e6bf7caea14f">
